e2e/app/page/create-account-page.ts

```diff
@@ -185,12 +176,11 @@ export default class CreateAccountPage extends BasePage {
 
   // Step 4: Fill out demographic survey information with default values
   async fillOutDemographicSurvey() {
-    const demograpicsSurveyPageHeader = 'Optional Demographics Survey';
-    await this.waitForTextExists(demograpicsSurveyPageHeader);
+    await waitUntilPageContainsText(this._page, 'Optional Demographics Survey');
     // Find and check on all checkboxes with same label: Prefer not to answer
     const targetXpath = '//*[normalize-space(text())="Prefer not to answer"]/ancestor::node()[1]/input[@type="checkbox"]';
-    await this.page.waitForXPath(targetXpath, { visible: true });
-    const checkboxes = await this.page.$x(targetXpath);
+    await this._page.waitForXPath(targetXpath, { visible: true });
+    const checkboxes = await this._page.$x(targetXpath);
     for (const ck of checkboxes) {
```

Comment:
I was curious and looked up how you could do these asynchronously just for fun. Something like:

```
await Promise.all(checkboxes.map(ck => ck.click()));
```